### ISSUE TITLE
Add menu entry to register EMF metamodel

### ISSUE THREAD

**Issue by RolandKluge** [eMoflon/emoflon-core#89]
Similar to 
http://www.kermeta.org/docs/html.chunked/KerMeta-Deploying-a-code/ch04.html
https://eclipse.org/epsilon/doc/articles/reflective-emf-tutorial/

**Comment by anthonyanjorin** [comment]
Hmm...  so we want to offer such functionality ourselves?  Why not just install and use the plugin from kermeta?

**Comment by RolandKluge** [comment]
This is true if we expect that we only need to register EPackages seldomly.

However, we may expect that, in the near future, eMoflon and Henshin will have more and more connection points, and Henshin sometimes requires to register EPackages (but offers no convenient menu entry for this, as far as I know). Therefore, I think that it's better if we ship this tiny feature with eMoflon instead of forcing people to install Kermeta (or Epsilon) for just this purpose.

**Comment by anthonyanjorin** [comment]
I agree!  And we can do it better too.  For example we could add an option:  "Automatically repeat this registration every time I start Eclipse", offering preferences where this of course can be changed later.  Kermeta is nice but reregistering the same metamodels 100 times was still a *** in the *** :)
